src/ALCProxy.Proxy/ProxyBuilder.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.Loader;
+using Microsoft.VisualBasic.CompilerServices;
+
+namespace ALCProxy
```

Comment:
We generally have tests for libraries in the tests directory with the same project name as the source but with `.Tests` added. Some thing to consider depending on the goal here. We also try to match the namespace name to the folder name.